pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -350,7 +350,7 @@
     "hlk_sw16" = ps: with ps; [ ]; # missing inputs: hlk-sw16
     "home_connect" = ps: with ps; [ aiohttp-cors ]; # missing inputs: homeconnect
     "homeassistant" = ps: with ps; [ ];
-    "homekit" = ps: with ps; [ HAP-python pyqrcode aiohttp-cors ha-ffmpeg zeroconf ]; # missing inputs: PyTurboJPEG base36 fnvhash
+    "homekit" = ps: with ps; [ HAP-python pyqrcode aiohttp-cors fnvhash ha-ffmpeg zeroconf ]; # missing inputs: PyTurboJPEG base36
```

Comment:
base36 is already in master.

Reply:
Yes, it was introduced with https://github.com/NixOS/nixpkgs/pull/109530